### ISSUE TITLE
Need Zoom feature for image (html/css/js)

### ISSUE THREAD

**Issue by vinodselvin** [vinodselvin/Kombu-Image-Previewer#4]
once image is opened, user can zoom in and zoom out image, without affecting pixels, also tell me how you are going to do it.

**Comment by lcosta-II2S** [comment]
Hey Vino!
I am trying to understand the requirement. On the gallery mode you want the user to be able to zoom in the image, correct? The pixelation of the image will depend entirely on the quality of the uploaded images, this is not inherent to the implementation since we are dealing with actual images and not vectors.
Looking forward to your reply

**Comment by vinodselvin** [comment]
@lcosta-II2S i agree with what you are saying, first we have to implement zoom feature, then we will look for pixelation and all improvements.

**Comment by vinodselvin** [comment]
UI part is done, flow has to be implemented in js.
